darts/ad/anomaly_model/forecasting_am.py

```diff
@@ -440,6 +440,7 @@ def show_anomalies(
         names_of_scorers: Union[str, Sequence[str]] = None,
         title: str = None,
         metric: Optional[Literal["AUC_ROC", "AUC_PR"]] = None,
+        multivariate_plot: bool = False,
```

Comment:
Here as well, an entry in the docstring would be great (should be identical to the one in anomaly model)